dtfs-central-api/src/v1/routes/utilisation-reports-routes.js

```diff
@@ -148,10 +148,10 @@ utilisationReportsRouter.route('/set-status').put(putUtilisationReportStatusCont
 
 /**
  * @openapi
- * /utilisation-reports/reconciliation-details/:reportId:
+ * /:utilisation-reports/reconciliation-details/:reportId:
```

Comment:
Surprised to see this when the other routes dont have a `:` - what does this do?

Reply:
It was just a typo 🙃